.github/workflows/publish.yml

```
@@ -28,6 +28,17 @@ jobs:
           rm -rf build/ dist/ *.egg-info
           python -m build
 
+      - name: Upload distributions to GitHub Release
+        uses: softprops/action-gh-release@v2
+        with:
+          files: dist/*
+          release_name: "Release ${{ github.ref_name }}"
```

actionlint (1.7.7)

input "release_name" is not defined in action "softprops/action-gh-release@v2". available inputs are "append_body", "body", "body_path", "discussion_category_name", "draft", "fail_on_unmatched_files", "files", "generate_release_notes", "make_latest", "name", "prerelease", "preserve_order", "repository", "tag_name", "target_commitish", "token"

(action)